test/integration/test_journal_entries.py

```diff
@@ -5,35 +5,39 @@
 
 logger = logging.getLogger(__name__)
 
-def test_get(nc):
```

Comment:
In this file:
- I split `test_get` into `test_get` & `test_get_all`.
- Moved both after `test_post`, so that the journal entry could be created first.